### DISCUSSION TITLE
Loss – ForBo7 // Salman Naqvi

### DISCUSSION THREAD

**Discussion by giscus** [ForBo7/forbo7.github.io#14] [Giscus Comments]
# Loss – ForBo7 // Salman Naqvi

A meaasure of performance of a model. It is used by the model to improve itself. Typically, the lower the loss, the better.

https://forbo7.github.io/dictionary/terms/loss.html

**Comment by slf188** [comment]
measure

**Reply by ForBo7** [reply]
Thank you for the correction!

**Reply by slf188** [reply]
Hey, I love your work! I'd like to add some more terms to the website! Kudos!

**Reply by ForBo7** [reply]
Sure, feel free to let me know what terms you would like to add! Would be good to email them; you can email me at [salmanaqvi+forblog@icloud.com](mailto:salmanaqvi+forblog@icloud.com).
